code/modules/power/supermatter/supermatter.dm

```diff
@@ -521,7 +521,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_shard)
 	dust_mob(user, cause = "hand")
 
 /obj/machinery/power/supermatter_shard/proc/dust_mob(mob/living/nom, vis_msg, mob_msg, cause)
-	if(nom.incorporeal_move)
+	if(nom.incorporeal_move || nom.status_flags & GODMODE)
```

Comment:
This is a good fix and all, but poly's ghost DOES have incorporeal_move set, so the fact that it gets past here bamboozles me

Reply:
Attack by doesn't call dust_mob, but consume instead

Reply:
why tf is poly attacking the shard? dumb bird

Reply:
Poly has engineering's best interests at heart, even in passing.